### PR TITLE
Add API contribution section to README.md

### PR DESCRIPTION
**Pull request by obfuscated-loop**
This adds a section to the README file with a simple explanation of what the API is and how to contribute.

### CODE REVIEW

**Comment by obfuscated-loop** [comment]
I'll start making PRs to that repo too - just wondering, have you got a brief setup guide for hosting it locally/in dev mode? I see that a token is needed, is the API all free etc?

**Comment by OhMyGuus** [comment]
> I'll start making PRs to that repo too - just wondering, have you got a brief setup guide for hosting it locally/in dev mode? I see that a token is needed, is the GitHub API all free etc?

The github api is all free to use you can get you're token from: 
https://github.com/settings/tokens

To add the token please use user-secrets, to make sure you don't push it by accident. 
On vscode you can use:
https://marketplace.visualstudio.com/items?itemName=adrianwilczynski.user-secrets
or using visual studio right click the project and then manage user secrets

after that a json will open and then u can just paste:
{
  "GithubConfiguration": {
    "RepoOwner": "OhMyGuus",
    "RepoName": "I-Still-Dont-Care-About-Cookies",
    "Token": "TOKENHERE"
  }
}


After u have changed that u can just use dotnet run, or just start it within visual studio (U might have to select https at the green > start button instead of docker)

File: README.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ This extension has been acquired by _**Avast**_ and I simply don't trust Avast w
 
 We welcome contributions to the translation of the extension. If you're interested in helping us translate the extension to your language, you can join us on [Crowdin](https://crowdin.com/project/i-still-dont-care-about-cookie/).
 
+## Contributing to The API
+This extension sends requests to an API hosted at *[api.istilldontcareaboutcookies.com](https://api.istilldontcareaboutcookies.com)* - while this is a backend-facing piece of software, the source code is available [here on GitHub](https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies-Api) and uses the C# ASP.NET library for an MVC structure.
```
Comment (OhMyGuus) [inline README.md:27]: can u change the * to _ (Same effect but prettier likes it more for some reason)

Comment (obfuscated-loop) [reply README.md]: Change made but she's still complaining!

**Pull request merged**